.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,33 @@
+name: Test_MacOS
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+    - 0.1.x
+    - 0.2.x
```

Comment:
Add this detail in the HOW-TO Release wiki page!

Reply:
I don't think we have such a page :(

Reply:
Open a respective issue about this 😕